test/end-to-end.js

```diff
@@ -29,6 +30,14 @@ var MYSQL_DATABASE = 'loopback_workspace_test';
 var MYSQL_USER = 'lbws';
 var MYSQL_PASSWORD = 'hbx42rec';
 
+// add conditional iterations
+it.skipIf = function(cond, name, fn) {
+  if (cond)
+    it.skip(name);
+  else
+    it(name, fn);
+};
```

Comment:
move to bdd helper like https://github.com/strongloop/loopback-datasource-juggler/blob/master/test/helpers/bdd-if.js